mongo/options/dboptions.go

```diff
@@ -13,44 +13,55 @@ import (
 	"go.mongodb.org/mongo-driver/mongo/writeconcern"
 )
 
-// DatabaseOptions represent all possible options to configure a Database.
+// DatabaseOptions represents options that can be used to configure a Database.
 type DatabaseOptions struct {
-	ReadConcern    *readconcern.ReadConcern   // The read concern for operations in the database.
-	WriteConcern   *writeconcern.WriteConcern // The write concern for operations in the database.
-	ReadPreference *readpref.ReadPref         // The read preference for operations in the database.
-	Registry       *bsoncodec.Registry        // The registry to be used to construct BSON encoders and decoders for the database.
+	// The read concern to use for operations executed on the database. The default value is nil, which means that
+	// the read concern of the client used to create the database will be used.
```

Comment:
"create" here too... Is create a term that's used in Go?  Or is there another term? In Java I would say "construct" and it would be clear that we're talking about Java, not the database.

Reply:
Changed to `client used to configure the Database` everywhere.